### DISCUSSION TITLE
Can I be a part of creating this project?

### DISCUSSION THREAD

**Discussion by sagar-monga** [lichess-org/mobile#9] [Q&A] [answered]
Hi devs, can I also be a part of the dev team for this app? 😄

**Comment by rf19l** [comment]
I'm also interested in contributing to the project. I assume once the project gets rolling there will be a contribution guideline similar to the current mobile app.
https://github.com/lichess-org/lichobile/blob/master/CONTRIBUTING.md

**Comment by semog** [comment]
I would also like to contribute as well. I read in the interview on lichess.org that you will be concentrating on puzzles first. This is a great area to start on. Having a development roadmap will help the rest of us know where to help out without overlapping the work of the core development team. I'm very glad you decided to use Flutter!

**Comment by timntorres** [comment]
I’m interested as well!

**Comment by luisgurmendez** [comment]
I'm also interested!!

**Comment by ghost** [comment]
I'd also like to help out anyway I can!

**Comment by RohanKhatua** [comment]
I would love to be able to contribute as well

**Comment by Nicola-Cestaro** [comment]
Me too!

**Comment by HDaoud** [comment]
Count me in please!

**Comment by veloce** [comment, marked as answer]
I'm glad you guys want to contribute. That's great! This is a big project and we'll certainly need help. 

As you can see the project is not even started yet, and you can understand contributions will not be open until the codebase has stabilized. 
Contrary to lichobile I'll work full time on this project, and so I will have higher quality/consistency expectations than before for myself and for anyone who want to contribute. All that to make sure the app will be the best one possible, and this project remains maintainable in the long run.

Therefore, here is what I estimate to be done before contributions are open (the list is not necessarily exhaustive):

- a first feature (puzzles) is fully done (code, tests, QA)
- I am satisfied with the code architecture and project structure
- internationalization and localization are set up
- accessibility requirements are implemented for the 1st feature
- routing is in place
- CI is installed and running
- contributions guidelines are written

**Comment by atharvashirdhankar** [comment]
I can contribute with CI/CD using Github Actions and Documentation part.
And also design work if needed

**Reply by prolificcoder** [reply]
Looks like you got volunteers for this already but second option if you'd like me to take a look at CI. Here is what i have done recently https://github.com/prolificcoder/house_stats/tree/master/.github and my linked in profile: https://www.linkedin.com/in/satyajitmalugu/

**Comment by malaschitz** [comment]
I also join in. I have experience with a chess app made in Flutter (online game, rest api, bishop and square library).

**Comment by severeduck** [comment]
An iOS developer here. I've been doing native development for years. Started with good old Objective-C, nowadays it's mostly Swift. I would like to get my hands dirty with Flutter, but more importantly to help the lichess community.

**Comment by kanjurer** [comment]
I am interested in contributing! I am only a student, but have software dev experience through internships. Lichess has given me a lot, so I'd like to give some back :)
